src/udisks2blockdevices.cpp

```diff
@@ -154,10 +154,7 @@ void BlockDevices::createBlockDevices(const QList<QDBusObjectPath> &devices)
 {
     QList<QDBusObjectPath> deviceMapperDevices;
     m_blockCount = devices.count();
-    if (m_blockCount == 0) {
-        m_populated = true;
-        emit externalStoragesPopulated();
-    }
+    updatePopulatedCheck();
```

Comment:
Maybe should be called like ...

Emit externalStoragePopulated from one place.